packages/editor/src/components/editor-canvas/index.js

```diff
@@ -40,6 +40,17 @@ const {
 
 const noop = () => {};
 
+/**
+ * These post types have a special editor where they don't allow you to fill the title
+ * and they don't apply the layout styles.
+ */
+const DESIGN_POST_TYPES = [
+	'wp_block',
+	'wp_template',
+	'wp_navigation',
+	'wp_template_part',
+];
```

Comment:
I'm fine with this fix, but of course I wonder if there's a systematic way to do it instead of using constants.